src/components/Routes.tsx

```diff
@@ -95,6 +96,24 @@ export function Routes() {
         )}
       />
 
+      <AuthenticatedRoute
+        path="/lab/imaging"
+        render={(routeProps) => (
+          <DataFetcher
+            key={routeProps.location.key}
+            dataFetcher={stanCore.GetRecordInPlaceInfo}
+          >
+            {(recordInPlaceInfo) => (
+              <RecordInPlace
+                title={"Imaging"}
+                operationType={"image"}
+                equipment={recordInPlaceInfo.equipments}
```

Comment:
I think you should only load equipment with the category "scanner" for the imaging page. That's what category was intended for.

Reply:
Thanks, good spot. Have changed it now.